coa-helm/values.yaml

```diff
@@ -77,3 +88,9 @@ cbas:
   name: cbas
   image: us.gcr.io/broad-dsp-gcr-public/cbas:0.0.18
 
+wds:
+  name: wds
+  image: gcr.io/broad-dsde-dev/workspacedata
+
+env:
+  swaggerBasePath: "SUPPLIED BY LEO"
```

Comment:
This weird little error sign means "no linebreak at the end of this file".

To make things easier when you `cat file`, I like to suggest keeping line breaks at the end of every file by default (there's also an intellij option that automatically adds one for you, so you don't even need to think about doing it)